IdentityCore/src/util/ios/MSIDKeychainUtil.m

```diff
@@ -22,30 +22,55 @@
 // THE SOFTWARE.
 
 #import "MSIDKeychainUtil.h"
+#import "MSIDKeychainUtil+Internal.h"
 
 @implementation MSIDKeychainUtil
 
 #pragma mark - Public
 
-+ (NSString *)teamId
+- (instancetype)init
 {
-    static dispatch_once_t once;
-    static NSString *keychainTeamId = nil;
+    return [[self class] sharedInstance];
```

Comment:
just declare init and new as unavailable instead

Reply:
Done